### ISSUE TITLE
train parameters ,lr, and loss

### ISSUE THREAD

**Issue by emilyemliyM** [edwardzhou130/PolarSeg#48]
Dear author ,I have run the model ,Could I know some details about the loss ,batchsize and learning rate initial value?

Reallt thanks!

**Comment by YangZhang4065** [comment]
Hi Emily,

All of our batch sizes and losses are either in the paper experiment section and/ or in this code repo. Is there a specific hyperparameter you want to know?
